android/src/main/java/com/doublesymmetry/trackplayer/service/MusicService.kt

```diff
@@ -420,39 +436,31 @@ class MusicService : HeadlessJsTaskService() {
     }
 
     override fun getTaskConfig(intent: Intent?): HeadlessJsTaskConfig {
-        return HeadlessJsTaskConfig(TASK_KEY, Arguments.createMap(), 0, true)
+        return HeadlessJsTaskConfig(TASK_KEY, Arguments.createMap(), TIMEOUT_MILLIS, true)
     }
 
-    override fun onHeadlessJsTaskFinish(taskId: Int) {
-        // Overridden to prevent the service from being terminated
```

Comment:
Ok, so we don't use this anymore to prevent the service from terminating? 🌟

Reply:
We use `START_STICKY` to prevent the service from terminating instead of this. The former is the proper way of keeping a service alive in Android.